mu-trees/addon/ember-config.js

```diff
@@ -25,11 +25,13 @@ export default function generateConfig(name) {
       'instance-initializers': { definitiveCollection: 'instance-initializer' },
       location: { definitiveCollection: 'main' },
       model: { definitiveCollection: 'models' },
+      modifier: { definitiveCollection: 'main' },
       'modifier-manager': { definitiveCollection: 'modifier-managers' },
       partial: { definitiveCollection: 'partials' },
       renderer: { definitiveCollection: 'main' },
       route: { definitiveCollection: 'routes' },
       router: { definitiveCollection: 'main' },
+      'route-map': { definitiveCollection: 'route-map' },
```

Comment:
I think this entry is incomplete 🤔, `route-map` as a collection doesn't exist below...